### ISSUE TITLE
Please add your adapter to the ioBroker repository

### ISSUE THREAD

**Issue by ldittmar81** [jlssmt/ioBroker.product-alert#84]
It would be really greate to see this adapter on ioBroker. Please add it to the repository. You can see how to do this here: https://github.com/ioBroker/ioBroker.repositories#add-a-new-adapter-to-the-latest-repository

Thanks for it :-)

**Comment by jlssmt** [comment]
@ldittmar81
Is this an automated message?
I'm not satisfied with the adapter yet. 
Why you want to see it in repo?

**Comment by ldittmar81** [comment]
Nope... this is not an automatic message. I would like to have my own Idealo. :-D

**Comment by jlssmt** [comment]
The adapter is currently very basic.
I have problems with some sites, like Mindfactory (i don't know why it's not working) and zalando (with different sizes of clothes).
But feel free to install it via github. A push to the repository is not necessary.

I also welcome every feedback, feature request, problem solution or code (via Pull Requests).
